cpp/src/gandiva/hash_utils.h

```diff
@@ -33,10 +33,14 @@ const char* gdv_hash_using_sha1(int64_t context, const void* message,
                                 size_t message_length, int32_t* out_length);
 
 GANDIVA_EXPORT
-const char* gdv_hash_using_sha(int64_t context, const void* message,
+const char* gdv_hash_using_ssl(int64_t context, const void* message,
```

Comment:
i think better name name it as gdv_hash_using_openssl(..) since ssl is just a library.
Also rename gdv_hash_using_md5 and gdv_hash_using_sha1 as gdv_md5_hash and gdv_sha1_hash

Reply:
Makes sense, already applied changes